### ISSUE TITLE
Task: Add time-aggregated loss for physical time evolution

### ISSUE THREAD

**Issue by evenmn** [ecmwf/WeatherGenerator#2712]
### Describe the task. It can be a feature, documentation, etc.

Producing outputs at a higher temporal frequency than the model’s time-window length now appears to work reasonably well. However, the resulting trajectories sometimes contain undesirable and physically unrealistic jumps within each time window.

This behaviour was observed when using a 6-hour time window with hourly outputs:

<img width="722" height="355" alt="Image" src="https://github.com/user-attachments/assets/4f3dda50-5608-4f69-9454-22eaa8e1d5db" />

A likely explanation is that the output states close to the 6-hour boundaries are more strongly constrained than those near the centre of the window. Consequently, the intermediate states may be less predictable and may evolve inconsistently.

A related problem is discussed in the recent HourGlass preprint, [arXiv:2607.11457](https://arxiv.org/pdf/2607.11457). The paper encourages temporal consistency by applying the training loss not only to the individual output states, but also to temporal statistics and consecutive differences.

### Hedgedoc URL, if you are keeping notes, plots, logs in hedgedoc.

https://gitlab.jsc.fz-juelich.de/hedgedoc/s/tzqiMi8vQ

### Area

- [ ] datasets, data readers, data preparation and transfer
- [x] model
- [x] science
- [ ] infrastructure and engineering
- [ ] evaluation, export and visualization
- [ ] documentation
- [ ] performance
